cpacker/MemGPT
# Fix bug with table listing for database connectors

**Please describe the purpose of this pull request.**
Tables are named with the prefix `memgpt_`, so we were removing the `memgpt_` string from the table name before returning it - however this did not consider the case where `memgpt_` was intended to be in the name. 

**How to test**
1. Load data: `memgpt load directory --input-files ./CONTRIBUTING.md --name MemGPT-Whitepaper`
2. List data to ensure correct name: `memgpt list sources`

**Have you tested this PR?**
Yes

**Related issues or PRs**
#535

**Is your PR over 500 lines of code?**
No
